### DISCUSSION TITLE
Placeholder for Current User

### DISCUSSION THREAD

**Discussion by hansvonn-halo** [jgraph/drawio#2687] [Ideas]
In LucidChart there was a text placeholder for the last modifiedby user. I see something similar is available for Confluence server version of the app per https://www.diagrams.net/doc/faq/predefined-placeholders.

Can we add a placeholder for the current user? Something like %username% to just show whoever is looking at the page? This would help when someone is making changes and exports the diagram. Ideally it would be the last modified user (outside of the confluence app), but the current user would be a good workaround! This is for the online version of the app.

This is a great tool. Thank you for the hard work!

**Comment by davidjgraph** [comment]
The problem is you can use the tool without logging in. What do you write there in that case?
